PayPalMessaging/src/main/java/com/braintreepayments/api/PayPalMessagingRequest.kt

```diff
@@ -0,0 +1,25 @@
+package com.braintreepayments.api
+
+/**
+ * Used to initialize a `BTPayPalMessagingView`
```

Comment:
```suggestion
 * Used to initialize a [PayPalMessagingView]
```
Bracket format to link to a class in Kotlin docs